src/main/java/pl/sdacademy/carrental/domain/rentals/Reservation.java

```diff
@@ -57,8 +57,8 @@
    @Column(name = "amount")
    private int payableAmount;
    
-   @AssertTrue
-   private boolean returnDateShouldBeAfterPickupDate() {
+   @AssertTrue(message = "Return date must be after pickup date")
+   private boolean isReturnAfterPickup() {
```

Comment:
might add nonull check maybe?

Reply:
validation added to entity class as well